docs/reference/sql/endpoints/odbc/configuration.asciidoc

```diff
@@ -172,7 +181,71 @@ will be considered by default. Choose _All Files (\*.*)_ from the drop down, if
 image:images/sql/odbc/dsn_editor_security_cert.png[]
 
 [float]
-===== 2.4 Logging parameters
+===== 2.4 Connection parameters
+The connection configuration can further be tweaked by the following parameters, but these should only need to be changed in rare cases.
+
+* Request timeout (s)
++
+The maximum number of seconds a request to the server can take. The value 0 disables disables the timeout.
+This corresponds to the `Timeout` setting in <<odbc-cfg-dsnparams>>.
++
+* Max page size (rows)
++
+The maximum number of rows that Elasticsearch SQL server should send the driver for one page.
+This corresponds to the `MaxFetchSize` setting in <<odbc-cfg-dsnparams>>.
++
+* Max page length (MB)
++
+The maximum number of megabytes that the driver will accept for one page.
+This corresponds to the `MaxBodySizeMB` setting in <<odbc-cfg-dsnparams>>.
++
+* Floats format
++
+How should the floating point numbers be printed, when these are converted to string by the driver.
+This corresponds to the `ScientificFloats` setting in <<odbc-cfg-dsnparams>>.
++
+* Data encoding
++
+How should the data between the server and the driver be encoded as.
+This corresponds to the `Packing` setting in <<odbc-cfg-dsnparams>>.
++
+* Data compression
++
+Should the data between the server and the driver be compressed?
+This corresponds to the `Compression` setting in <<odbc-cfg-dsnparams>>.
++
+* Follow HTTP redirects?
```

Comment:
I don't thing the `?` is necessary in the header.
Same for the rest of the headers.